tests/expressions/tests.py

```diff
@@ -1450,7 +1450,10 @@ def test_functions(self):
         self.assertEqual(repr(Concat('a', 'b')), "Concat(ConcatPair(F(a), F(b)))")
         self.assertEqual(repr(Length('a')), "Length(F(a))")
         self.assertEqual(repr(Lower('a')), "Lower(F(a))")
+        self.assertEqual(repr(LTrim('a')), 'LTrim(F(a))')
```

Comment:
I'm not sure these are needed since `__repr__()` isn't overridden. What's your thinking?

Reply:
Agreed.